snippets/SMTP/smtp_example.js

```diff
@@ -0,0 +1,30 @@
+// This example utilizes the 'nodemailer' and 'nodemailer-smtp-transport'
+// modules which can be installed using the following commands:
+// npm install nodemailer
+// npm install nodemailer-smtp-transport
+ 
```

Comment:
Maybe also put a note in about how a user could opt for https://github.com/SparkPost/nodemailer-sparkpost-transport as well?